packages/sdk-react/src/components/auth/Auth.tsx

```diff
@@ -60,7 +64,7 @@ interface AuthProps {
     facebookEnabled: boolean;
     googleEnabled: boolean;
   };
-  configOrder: string[];
+  configOrder: AuthSection[];
```

Comment:
let's make these changes in another PR - this breaks example build since this type is not exported and will not be consistent with our docs at the moment